### PR TITLE
genre is changed into a dropdown

### PR DESCRIPTION
**Pull request by mounicap6**

### CODE REVIEW

File: src/main/java/com.amazon.bookstore/BookstoreController.java [hunk 1]
```diff
@@ -52,12 +52,6 @@ public String mainpage(Model model) {
         return "ownerBookstore";
     }
 
-    @PostMapping("/editBookstore")
```
Comment (SoniaHL) [inline src/main/java/com.amazon.bookstore/BookstoreController.java:55]: You're still calling /editBookstore in the ownerBookstore.html so removing this causes an error

**Review by SoniaHL — CHANGES_REQUESTED** [review]
Make the change in ownerBookstore.html before it is merged

File: src/main/java/com.amazon.bookstore/BookstoreController.java (hunk 1 above)
Comment (mounicap6) [reply src/main/java/com.amazon.bookstore/BookstoreController.java:55]: fixed!

Comment (mounicap6) [reply src/main/java/com.amazon.bookstore/BookstoreController.java:55]: added it back, Lynn will be using this.

**Review by SoniaHL — APPROVED** [review]

**Review by sarahlamonica — APPROVED** [review]

**Pull request merged**